Architect-Coders/MU8
# Javier/feature/comics unit tests

What's in:

- Created tests for ComicsViewModel.

Testing Done:

- Android Studio

How it looks:

![testjavier](https://user-images.githubusercontent.com/10672408/81461534-20c06380-9172-11ea-836d-dc38e9b463d6.png)

How to test:

- Launch the app and navigate through it. Verify is working normally.

- Go to the EventsViewModelTest and execute it, you should see all tests passing.

Signed-off-by:

- Javier Cardona javier.mcardona@gmail.com
